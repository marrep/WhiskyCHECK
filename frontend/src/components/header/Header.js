import Logo from "../../assets/icons/logo.svg";
import React from "react";
import styled from "styled-components/macro";

export default function Header() {
  return (
    <MainWrapper>
      <img src={Logo} alt="WhiskyCheck Logo" data-testid="Logo" />
    </MainWrapper>
  );
}

const MainWrapper = styled.div`
  align-items: center;
  background-color: #003f8a;
  display: flex;
  flex-direction: column;
  height: 10vh;
  justify-content: center;
  width: 100%;
`;
