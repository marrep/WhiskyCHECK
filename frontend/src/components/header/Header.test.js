import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header", () => {
  it("contains an image", () => {
    const { container, getByAltText } = render(<Header></Header>);
    expect(container.firstChild).toMatchSnapshot();
    expect(getByAltText("WhiskyCheck Logo")).toBeTruthy();
  });
});
