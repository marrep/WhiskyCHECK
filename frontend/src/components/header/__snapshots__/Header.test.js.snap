// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header contains an image 1`] = `
<div
  class="sc-bdfBwQ ibSdKW"
>
  <img
    alt="WhiskyCheck Logo"
    data-testid="Logo"
    src="logo.svg"
  />
</div>
`;
